plugin/build_baron.go

```diff
@@ -17,114 +17,11 @@ func init() {
 	Publish(&BuildBaronPlugin{})
 }
 
-type bbPluginOptions struct {
-	Projects map[string]evergreen.BuildBaronSettings
-}
-
-type BuildBaronPlugin struct {
-	opts *bbPluginOptions
-}
+type BuildBaronPlugin struct{}
```

Comment:
Is there a reason we can't just delete this file completely? (Not sure if it breaks something but)

Reply:
The struct needs to exist for the Publish method at the top of each plugin file to register the plugin (as we do for manifest, perf, BB, etc).  This is the way we register plugins so that their associated PanelConfig can be displayed in the UI.  This can probably have its pattern change as an improvement in the future since it is a bit confusing but it would be out of scope for this ticket.  See the RegisterPlugins method for reference.